Source/PINDiskCache.m

```diff
@@ -129,6 +142,12 @@ - (instancetype)initWithName:(NSString *)name
         return nil;
     }
     
+    if ((keyEncoder && !keyDecoder) ||
```

Comment:
I made like in `serializer/deserializer` case above. Should I change both to `NSAssert` ?